datafusion/common/Cargo.toml

```diff
@@ -36,7 +36,6 @@ name = "datafusion_common"
 path = "src/lib.rs"
 
 [features]
-default = ["recursive-protection"]
```

Comment:
having the feature enabled by default in subcrates meant I think it basically was not possible to disable when using datafusion core crate